dcos/dcos-ui
# DCOS-49183: [SOAK] Plans Tab: URIError: URI malformed

Add a check if service id and scheduler task id
are valid URI and don't display scheduler logs if not.

Closes https://jira.mesosphere.com/browse/DCOS-49183

## Testing
1. In `dcos-ui/plugins/services/src/js/components/SDKPlansTab.tsx` change line 79 to:
```
    let { service, schedulerTaskId } = this.props;
    schedulerTaskId = "\uD800";
```
2. Start a service that support SDK plans, for example kafka.
3. Go to plans tab and see that it works and has no "view scheduler logs" button.

## Trade-offs
I tried to reproduce the error, but couldn't, so mocking it is the next best thing.

## Dependencies
None.

## Screenshots
### Before
![Screenshot from 2019-03-22 14-02-29](https://user-images.githubusercontent.com/40791275/54822741-4492c280-4cae-11e9-855b-3bb887eabc38.png)

### After
![Screenshot from 2019-03-22 14-03-12](https://user-images.githubusercontent.com/40791275/54822748-478db300-4cae-11e9-8961-a1ff235bb176.png)
